src/implementations/twig/storybook/package.json

```diff
@@ -3,7 +3,7 @@
   "name": "@ecl/ec-twig-storybook",
   "author": "European Commission",
   "license": "EUPL-1.1",
-  "version": "2.31.1",
+  "version": "2.33.0",
```

Comment:
I guess we should put it to `2.34` already

Reply:
Good point. I did a step forward on this: set all packages to "private" and version 3.0.0. A few unnecessary deps got dropped during this update.